stack-nightly.yaml

```diff
@@ -9,15 +9,10 @@ nix:
   packages:
     - zlib
     - http-client-tls-0.3.4
-flags:
-  mintty:
-    win32-2-5: false
 extra-deps:
-- Cabal-2.0.0.2
-- mintty-0.1.1
-- text-metrics-0.3.0
 - bindings-uname-0.1
-- path-0.6.1
-- path-io-1.3.3
-- extra-1.6
-- monad-logger-0.3.25
+- hackage-security-0.5.2.2
+- store-0.4.3.2
```

Comment:
`store` and `store-core` got back into the nightlies (as of https://github.com/fpco/stackage/pull/2845):
  * https://www.stackage.org/nightly-2017-09-10/package/store-0.4.3.2
  * https://www.stackage.org/nightly-2017-09-10/package/store-core-0.4.1